datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -97,11 +224,53 @@ export default function SchemaHeader({
                     )}
                     {maxVersion > 0 &&
                         (editMode ? (
-                            <ShowVersionButton onClick={() => setEditMode?.(false)}>Version History</ShowVersionButton>
+                            <ShowVersionButton onClick={() => setEditMode?.(false)}>Version Blame</ShowVersionButton>
                         ) : (
                             <ShowVersionButton onClick={() => setEditMode?.(true)}>Back</ShowVersionButton>
                         ))}
-                </div>
+                </LeftButtonsGroup>
+                <RightButtonsGroup>
+                    <CurrentVersionTimestampText>{lastUpdatedTimeString}</CurrentVersionTimestampText>
+                    <BlameRadio value={schemaView} onChange={onSchemaBlameToggle}>
+                        <BlameRadioButton value={SchemaViewType.NORMAL} data-testid="schema-normal-button">
+                            Normal
+                        </BlameRadioButton>
+                        <BlameRadioButton value={SchemaViewType.BLAME} data-testid="schema-blame-button">
+                            Blame
+                        </BlameRadioButton>
+                    </BlameRadio>
+                    <SchemaBlameSelector
+                        value={selectedVersion}
+                        onChange={(e) => {
+                            const datasetVersion: string = e as string;
+                            navigateToVersionedDatasetUrl({
+                                location,
+                                history,
+                                datasetVersion,
+                            });
+                        }}
+                        data-testid="schema-version-selector-dropdown"
+                        suffixIcon={<StyledCaretDownOutlined />}
+                    >
+                        {renderOptions()}
+                    </SchemaBlameSelector>
+                    <Popover
+                        overlayStyle={{ maxWidth: 240 }}
+                        placement="right"
+                        content={
+                            <div>
+                                Semantic versions for this view were computed using TECHNICAL_SCHEMA. You can find more
```

Comment:
Minor nit: you can say "were computed using the Technical Schema". You don't need to put raw details like TECHNICAL_SCHEMA in there - most users won't know why this is formatted like this

Reply:
Makes sense